tests/test_plotting.py

```diff
@@ -0,0 +1,335 @@
+import unittest
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+
+from viroconcom.read_write import read_dataset, read_contour, write_contour
+from viroconcom.plot import plot_contour, plot_wave_breaking_limit, \
+    plot_marginal_fit, plot_dependence_functions, plot_confidence_interval, \
+    PlottedSample
+from viroconcom.contour_statistics import points_outside, \
+    sort_points_to_form_continous_line
+
+from viroconcom.params import ConstantParam, FunctionParam
+from viroconcom.distributions import LognormalDistribution, WeibullDistribution, \
+    MultivariateDistribution
+from viroconcom.fitting import Fit
+from viroconcom.contours import IFormContour, HighestDensityContour
+
+
+class ReadWriteTest(unittest.TestCase):
+
+    def test_read_dataset(self):
+        """
+        Reads the provided dataset.
+        """
+        sample_hs, sample_tz, label_hs, label_tz = read_dataset()
+        self.assertAlmostEqual(sample_hs[0], 0.2845, delta=0.00001)
+
+    def test_read_write_contour(self):

```

Comment:
What would be the benefit? Also, I like it that the test uses the function as a user would do (and does not use a special tempfile).

Reply:
If an error occurs during test execution the os.remove might no be called. In a tmp_dir this would not matter as the file will be deleted by the operation system. If using the context manager or a pytest fixture python would make sure the file is deleted. Also the user probably would not use os.remove to delete a file.

Reply:
Okay agree. I would nevertheless like to keep it as is to move forward with merging :-)